Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Build stage: Install python dependencies
 # ===
 FROM ubuntu:focal AS python-dependencies
-RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools python3-wheel build-essential
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools python3-wheel build-essential git
```

Comment:
Is this just so the demo will spin up with the `git` line in the `requirements.txt`?

That's fine, but please make sure we remove this again if we don't need it for any other reason.